js/components/datatable/row.vue

```diff
@@ -47,9 +46,9 @@ export default {
          *                             in the application scope when
          *                             the view is loaded
          */
-        load_cell: function(name) {
+        load_cell(name) {
             if (!this.$options.components.hasOwnProperty(name)) {
-                let options = require('./cells/' + name + '.vue');
+                const options = require('./cells/' + name + '.vue');
```

Comment:
Not in this case: webpack 1.x dynamic import only works on ES5. I added a comment with references.